modules/ROOT/pages/configure-third-party.adoc

```diff
@@ -1,6 +1,6 @@
 = Configuring and Managing Third-Party Connections
 
-If your trading partners use third-party service providers such as VANs or 3PLs (third-party logistics providers) who manage the B2B connectivity with your organization on behalf of your partners, you can create a third-party connection to manage configurations such as the endpoints, identifiers, and certificates of the third-party provider.
+If your partners use third-party service providers such as VANs or 3PLs (third-party logistics providers) who manage the B2B connectivity with your organization on behalf of your partners, you can create a third-party connection to manage configurations such as the endpoints, identifiers, and certificates of the third-party provider.
```

Comment:
Is vale OK with this sentence length?

Reply:
No, but I am not sure how to split it up.